references.bib

```diff
@@ -223,12 +231,18 @@ @article{sankappanavar1981course
     year      = {1981},
     publisher = {Citeseer}
   }
-  @book{russell2020principles,
-    title     = {The Principles Of Mathematics},
-    author    = {Russell, Bertrand},
-    year      = {2020},
-    publisher = {Routledge}
+
+  @article{russell2020principles,
```

Comment:
This citation is definitely not right - the author needs to be Bertrand Russell! What was wrong with the previous one, which seemed much better.